### ISSUE TITLE
AttributeError: module 'dlib' has no attribute 'load_rgb_image'

### ISSUE THREAD

**Issue by luckynote** [davisking/dlib#1312]
ubuntu@ubuntu:~/face-recognition/dlib/python_examples$ python face_detector.py ../examples/faces/2007_007763.jpg 
Processing file: ../examples/faces/2007_007763.jpg
Traceback (most recent call last):
  File "face_detector.py", line 53, in <module>
    img = dlib.load_rgb_image(f)
AttributeError: module 'dlib' has no attribute 'load_rgb_image'

/home/ubuntu/anaconda3/bin/python /home/ubuntu/face-recognition/dlib/python_examples/face_recognition.py
Traceback (most recent call last):
Processing file: ../examples/faces/2007_007763.jpg
  File "/home/ubuntu/face-recognition/dlib/python_examples/face_recognition.py", line 82, in <module>
    img = dlib.load_rgb_image(f)
AttributeError: module 'dlib' has no attribute 'load_rgb_image'

Dlib is installed ！ When  I use example_python，I meet the question. It is showing AttributeError: module 'dlib' has no attribute 'load_rgb_image' at pycharm. However, It is not this error by c++ in qt creator.  I don't kown how to repair. 

* **Version**: 19.10
* **Where did you get dlib**:  github：https://github.com/davisking/dlib
* **Platform**: ubuntu 16.04 x64
* **Compiler**:  pycharm-community-2017.3.3  Anaconda3 python3.6

**Comment by yyf1986** [comment]
I have the same problem

**Comment by davisking** [comment]
If you get this error then it means you are using an old version of dlib.  Get the new one from git.  I see that you claimed you did get the new one from git, but considering that this function is missing you clearly are using an old version of dlib.

**Comment by Kostanos** [comment]
I have the same error, the docker image I compiled with this instruction:
```
RUN cd ~ && \
    git clone --quiet --depth 1 https://github.com/davisking/dlib.git dlib/ && \
    cd  dlib/ && \
    python setup.py install --yes USE_AVX_INSTRUCTIONS
```

So, I assume I got the latest version from git (master branch), I compiled it few days ago. Anyone resolved this?
Do I need to download some branch/tag instead?

**Comment by Kostanos** [comment]
Ok, after checking, I see the version is: `19.7.99`,
So yes, it is not a latest one.
But this one that I build from master of the repo. Any suggestions?

**Comment by Yolanda-Gao** [comment]
Is the problem solved? I am using a '19.16.0' version dlib but still cannot load some functions.

**Comment by Yolanda-Gao** [comment]
Oh it works now... I made a silly mistake. Sorry about it!

**Comment by Mertkarakoc1** [comment]
> Oh it works now... I made a silly mistake. Sorry about it! And thank you very much for reply!

Can u tell me the mistake u did? Cus I have the same error.

**Comment by rajan3010** [comment]
Hi Iam getting the same error and for dlib.__version__ too . Any ideas on how to solve it ?

**Comment by gitlGl** [comment]
Is the problem solved? I am using a '19.8.1' version dlib but still cannot load some functions.
